rdmd_test.d

```diff
@@ -310,6 +310,16 @@ void runTests()
         assert(res.status == -SIGSEGV, format("%s", res));
     }
 
+    /* https://issues.dlang.org/show_bug.cgi?id=11997- rdmd should search it's
+    binary path for the compiler */
+    string localDMD = buildPath(tempDir(), baseName(compiler));
+    std.file.write(localDMD, "empty shell");
+    scope(exit) std.file.remove(localDMD);
+
+    res = execute(rdmdApp ~ [modelSwitch, "--force", "--chatty", "--eval=writeln(`Compiler found.`);"]);
+    assert(res.status == 1, res.output);
+    assert(res.output.canFind(`spawn ["` ~ localDMD ~ `",`));
```

Comment:
> Wouldn't dmd need druntime and phobos and a config file too? Becomes too much of a PITA, especially once you consider other compilers, like ldc.

Not if the dmd binary in question is just a small stub program that prints something and returns success OSLT.